lib/xcpretty/formatters/simple.rb

```diff
@@ -39,6 +39,10 @@ def format_compile_xib(file_name, file_path)
       format("Compiling", file_name)
     end
 
+    def format_compile_storyboard(file_name, file_path)
+      format("Compiling", file_name)
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.